scripts/ci/jobs/check-konflux-pipelines.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# This script is to ensure that modifications to our Konflux pipelines follow our expectations and conventions.
+# This script is intended to be run in CI
+
+set -euo pipefail
+
+ensure_create_snapshot_runs_last() {
+    expected_runafter="$(yq eval '.spec.tasks[] | select(.name != "create-acs-style-snapshot") | .name' .tekton/operator-bundle-pipeline.yaml | sort)"
+    actual_runafter="$(yq eval '.spec.tasks[] | select(.name == "create-acs-style-snapshot") | .runAfter[]' .tekton/operator-bundle-pipeline.yaml)"
+
+    if ! DIFF=$(diff <(echo "${expected_runafter}") <(echo "${actual_runafter}")); then
```

Comment:
1. With my suggestion below it will become unnecessary to capture diff in a variable. On the opposite, it can be printed as-is.
2. Also, I find it more useful to work with two-column diff and I find the default output hard to comprehend.
3. After that, it may be appropriate to tell what we're checking.

```suggestion
    echo "Checking .tekton/operator-bundle-pipeline.yaml that create-acs-style-snapshot task's runAfter contents match the expected ones (left - expected, right - actual):"
    # Please note that diff won't be printed if there's no diff, so the above message surely needs tweaking.
    if ! diff --side-by-side <(echo "${expected_runafter}") <(echo "${actual_runafter}"); then
```